shell/platform/darwin/macos/framework/Source/FlutterBackingStoreData.mm

```diff
@@ -12,11 +12,13 @@
 @implementation FlutterBackingStoreData
 
 - (nullable instancetype)initWithLayerId:(size_t)layerId
-                              fbProvider:(nonnull FlutterFrameBufferProvider*)fbProvider
+                              fbProvider:(nullable FlutterFrameBufferProvider*)fbProvider
                          ioSurfaceHolder:(nonnull FlutterIOSurfaceHolder*)ioSurfaceHolder {
   if (self = [super init]) {
     _layerId = layerId;
-    _frameBufferProvider = fbProvider;
```

Comment:
why does this matter? `_frameBufferProvider` can always be `fbProvider` without the null check right?

Reply:
yeah it doesn't matter, just habit.